### ISSUE TITLE
cdn_repo: handle reading packages with no tags

### ISSUE THREAD

**Issue by ktdreyer** [ktdreyer/errata-tool-ansible#72]
The `get_package_tags()` method returns a dictionary of package names and tags for a repository. If a package in the repository has zero tags, this method will not return any key for that package name at all. The reason is that the `/api/v1/cdn_repo_package_tags` API endpoint does not return any entries for packages with zero tags.

Later in the `ensure_packages_tags()` method, we hit a `KeyError` early when we try to compare the `current_tags` for a package with the `desired_tags`, because `current_tags` has no key for this `package_name`.

We need to update `ensure_package_tags()` to handle the missing `package_name` keys in the `current_tags` data we obtain from `get_package_tags()`.

**Comment by ktdreyer** [comment]
Steps to reproduce:

1. Create a CDN repo with a package with no tags
2. Try to read the CDN repo again with `errata_tool_cdn_repo`

The following patch fixes this for me:

```diff
diff --git a/library/errata_tool_cdn_repo.py b/library/errata_tool_cdn_repo.py
index 739ea59..31c8b85 100644
--- a/library/errata_tool_cdn_repo.py
+++ b/library/errata_tool_cdn_repo.py
@@ -479,7 +481,7 @@ def ensure_packages_tags(client, name, check_mode, packages):
     current = get_package_tags(client, name)
 
     for package_name in packages:
-        current_tags = current[package_name]
+        current_tags = current.get(package_name, [])
         desired_tags = packages[package_name]
 
         package_changes = ensure_package_tags(client,
```

Still need to update some docstrings and write unit tests for this condition.

**Comment by ktdreyer** [comment]
This is resolved in b8b96c7df9aaf5b1c1f6213d9e7bf6ae31ef0f25, uploaded to [Ansible Galaxy](https://galaxy.ansible.com/ktdreyer/errata_tool_ansible) as version 0.0.86.
